main.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
Don't you want to move this file to cmd/main.go?

Reply:
In my opinion, the main package is placed in the root of the repository if the program itself is implemented as a binary. If it is implemented as a library, its main code is placed in the root, and the package implementing it is placed in cmd.

Here we create a binary file of the program, so placing it in cmd makes no sense.